### PR TITLE
Create entire frontend

### PR DESCRIPTION
**Pull request by antingle**
Make sure `npm install` inside the frontend folder before running. Otherwise, entire frontend created

### CODE REVIEW

**Review by NachoNoCheese — APPROVED** [review]
thank you very cool

File: frontend/src/assets/cash4life_logo.png
Comment (stollerdaws) [inline frontend/src/assets/cash4life_logo.png:1]: yeah right, get real

**Review by ColeWarren — APPROVED** [review]
The logic is as brilliant as Einstein himself.

**Review by stollerdaws — APPROVED** [review]
looks good, not real

**Review by Azure-Agst — APPROVED** [review]
:)

**Comment by antingle** [comment]
wait guys are you sure

**Comment by Azure-Agst** [comment]
yes :)

**Pull request merged**